theia-cpp-docker/Dockerfile

```diff
@@ -97,11 +97,12 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
     echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main" > /etc/apt/sources.list.d/llvm.list && \
     apt-get update && \
     apt-get install -y cmake \
-                       clang-tools-9 \
-                       clang-tidy-9 \
+                       clang-tools-10 \
+                       clangd-10 \
+                       clang-tidy-10 \
                        gdb && \
-    ln -s /usr/bin/clangd-9 /usr/bin/clangd && \
-    ln -s /usr/bin/clang-tidy-9 /usr/bin/clang-tidy && \
+    ln -s /usr/bin/clangd-10 /usr/bin/clangd && \
```

Comment:
I don't believe this is part of your intended changes, the change is present in #232

Reply:
Ok

Reply:
Hmm, I did a quick test. --depth 2 is required:

```
git clone --depth 1 git@github.com:theia-ide/theia-apps.git
cd theia-apps
$ git diff HEAD HEAD~1
fatal: ambiguous argument 'HEAD~1': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

With "-depth 2" this command works as expected. A depth of one ensures that the HEAD is available, a depth of 2 also ensures that we can compare with the previous commit.